### ISSUE TITLE
Web support 

### ISSUE THREAD

**Issue by brt-manishp** [memspace/zefyr#442]
I am try to run Zefyr on Web version but i am getting so many error, is there any plans for Web support or have any alternative. Please suggest.

Thank you

**Comment by cgestes** [comment]
Zefyr cannot work flowlessly on the web before at least this lands on flutter side: https://github.com/flutter/flutter/issues/55587

**Comment by jbryanh** [comment]
Looks like they are actively working on it.  I'm curious though: I have successfully loaded this, and it works, with the example provided.  However, any double clicks or long clicks, or attempts to select text get an addListener error.  I'm trying to unwind how to implement the selectionControls TextSelectionControls option, however, I'm not sure if this the route to take.  Any guidance?

**Comment by singerdmx** [comment]
Could you paste error message?

**Comment by jbryanh** [comment]
======== Exception caught by widgets library =======================================================
The following JSNoSuchMethodError was thrown building _SelectionToolbar(state: _SelectionToolbarState#3038c):
NoSuchMethodError: invalid member on null: 'addListener'

The relevant error-causing widget was: 
  Expanded file:///Users/.../lib/views/layout_template/layout_template.dart:39:17
When the exception was thrown, this was the stack: 
packages/flutter/src/material/text_selection.dart 168:28  initState
packages/flutter/src/widgets/framework.dart 4762:57       [_firstBuild]
packages/flutter/src/widgets/framework.dart 4599:5        mount
packages/flutter/src/widgets/framework.dart 3541:13       inflateWidget
packages/flutter/src/widgets/framework.dart 3306:18       updateChild
...
====================================================================================================

**Comment by dannnnthemannnn** [comment]
I too am getting this error

**Comment by singerdmx** [comment]
@dannnnthemannnn 

This repo has been inactive for months.  See https://github.com/memspace/zefyr/issues/409#issuecomment-790671992

Just FYI to save your time here.

**Comment by dannnnthemannnn** [comment]
Yeah, I know but I was hoping someone was maintaining a fork or something. 

The flutter-quill is not working on web very well currently and this seems to be quite good on web so was hoping to just use zefyr until the flutter-quill web version is farther along.

**Comment by pratikpparikh** [comment]
@dannnnthemannnn  and @singerdmx is flutter-quill better or zefyr to add web support too?
